src/Common/Parameters.cpp

```diff
@@ -520,6 +521,23 @@ void Parameters::SetDefaultValues(void)
   //-----------------------------------------------------------------------------------------------
   stringparams["supernova_feedback"] = "none";
 
+  // Radiative feedback parameters
+  //-----------------------------------------------------------------------------------------------
+  intparams["rad_fb"] = 0;
```

Comment:
Could perhaps all new parameters here (including the 'lombardi_method' one above) be added to the userguide?

Reply:
Done, let me know if I forgot anything!